packages/victory-brush-container/src/victory-brush-container.js

```diff
@@ -20,7 +20,7 @@ export const brushContainerMixin = (base) =>
         y: PropTypes.array
       }),
       brushStyle: PropTypes.object,
-      defaultBrushArea: PropTypes.oneOf(["all", "disable", "none"]),
+      defaultBrushArea: PropTypes.oneOf(["all", "disable", "none", "move"]),
```

Comment:
This addition is making me want to rename this prop 😅
I'm going to avoid the breaking change for now so I can get this feature in without a major version bump. For the future, I think this should be something more like `brushClearing: PropTypes.oneOf(["selectAll", "disable", "selectNone", "move"])` Open to suggestions on what would be clearest. What do you think?

Reply:
Yeah, I had basically the same thought. Maybe I can add it as an alias for now?